### PR TITLE
Qpd 388 build

### PR DESCRIPTION
**Pull request by manoj016**

### CODE REVIEW

**Review by shivinquartic — APPROVED** [review]
lgtm

**Comment by shubham-quartic** [comment]
Wouldn't this have conflict with tensorflow from same repository?
```tensorflow 2.5.0 requires six~=1.15.0, but you have six 1.16.0 which is incompatible.```

File: requirements.txt [hunk 1]
```diff
@@ -37,7 +37,7 @@ pytz==2021.1
 PyYAML==5.4.1
 regex==2020.11.13
 requests==2.25.1
-six==1.15.0
```
Comment (shubham-quartic) [inline requirements.txt:40]: Wouldn't this have conflict with tensorflow from same repository?
```tensorflow 2.5.0 requires six~=1.15.0, but you have six 1.16.0 which is incompatible.```

**Review by shubham-quartic — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (shubham-quartic) [reply requirements.txt:40]: Also numpy, why is that bumped?
```  - tensorflow==2.5.0 [requires: numpy~=1.19.2]```

**Comment by manoj016** [comment]
Closing this as its not required.

**Pull request closed without merging**